src/components/ProjectSettingsTabs/GithubCommitQueueTab/getFormSchema.ts

```diff
@@ -248,6 +258,16 @@ export const getFormSchema = (
             },
           },
         },
+        githubTriggerAliases: {
```

Comment:
Is there something we can do to make it more obvious to the user that these are hoverable?

Reply:
Good idea, I think the [inline definition](https://www.mongodb.design/component/inline-definition/) component makes sense here!
<img width="426" alt="image" src="https://user-images.githubusercontent.com/9298431/159516119-2722b293-e519-4914-aadb-9ab7f1f8cbbc.png">